DESCRIPTION

```diff
@@ -22,19 +24,21 @@ Copyright: This software is in the public domain because it contains materials
     http://www.usgs.gov/visual-id/credit_usgs.html#copyright
 Depends:
     R (>= 3.0),
-    GLMr,
-    rLakeAnalyzer (>= 1.8.3)
+    GLM3r,
```

Comment:
Can/should we remove `GLMr` and `GLM3r` from depends and put both in `suggests`? That way you could use whichever GLM version you want

Reply:
Hmmm. Well the package actually depends on GLM3r to run example sims. It does not depend on GLMr (which is not maintained). So my vote is to keep as is.

Reply:
Ok, but it creates some challenges to people that simply want to use glmtools on .nc files. You _have_ to install GLM3r to use this package. I'm also thinking about my own use on HPC, where installations are a pain so I want to keep things as bare as possible. But perhaps that isn't a common situation.

Reply:
So I just kept the package the same as it had been. It required `GLMr` to execute `run_example_sim` and run the examples. We don't have to do this. We could remove all `run_example_sim` examples, and just have an ext .nc file that examples run from. 

By doing that it would not require `GLM3r`